conf/application.conf

```diff
@@ -1,12 +1,14 @@
 # https://www.playframework.com/documentation/latest/Configuration
 play.http.port=9000
 
-auth.url="http://localhost:8080"
+auth.url="http://localhost:8081"
 auth.callback="http://localhost:9000/callback"
-auth.secret="secretValue"
+auth.secret="set_me"
```

Comment:
We changed this away from "set_me" because it looked like you were supposed to override it in this file. Maybe "placeholderSecretValue" would be clearer?

Reply:
Yep, that makes sense, I've changed it to placeholderSecretValue